latexify/core.py

```diff
@@ -224,6 +244,27 @@ def visit_If(self, node, action):  # pylint: disable=invalid-name
     latex += self.visit(node)
     return latex + r', & \mathrm{otherwise} \end{array} \right.'
 
+  def visit_GeneratorExp_sum(self, node):  # pylint: disable=invalid-name
```

Comment:
This could have been implemented with generic `visit_GeneratorExp`, which might be useful for FRs in the future (ditto for `visit_comprehension'. In that case, we might need some checks in `decorated_lstr_and_arg`.

You could view the generic implementation in this commit (sorry it's a bit dirty before refactoring).
https://github.com/google/latexify_py/pull/32/commits/0c5d67fa8aa44064b74c711a7d567d11953877ec

Reply:
What are FRs?

This function can be reused with other operators (e.g., products and integrals have the same syntax). We can use some general action name rather than `sum`.

Reply:
I think the bare visitor (=visitor with `None` action) of every node should return only a LaTeX string. Otherwise every parent node is required to know what kind of objects are returned from children and how to process them to construct LaTeX strings.

Reply:
> What are FRs?
Sorry for not being clear, but I meant feature requests. I thought some operations could use `GeneratorExp` and there might not be a reason to limit my function only to summation.

However, your point on bare visitors makes sense.

Reply:
> This function can be reused with other operators (e.g., products and integrals have the same syntax). We can use some general action name rather than sum.

`set_bounds` as suggested in the new commit?